enterprise/coderd/provisionerkeys.go

```diff
@@ -89,7 +106,7 @@ func (api *API) provisionerKeys(rw http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	organization := httpmw.OrganizationParam(r)
 
-	pks, err := api.Database.ListProvisionerKeysByOrganization(ctx, organization.ID)
+	pks, err := api.Database.ListProvisionerKeysByOrganizationExcludeReserved(ctx, organization.ID)
```

Comment:
Why does the api exclude reserved? Why not include psk and built ins?

Reply:
This endpoint is used by the CLI and I don't think it's necessary to show them in isolation or without a reference to the daemon counterparts since they are more placeholder.

Reply:
Do you think we can deprecate the old endpoint at some point in the future? Should we mark it deprecated in swagger?